includes/general.php

```diff
@@ -972,15 +972,15 @@ function pods_shortcode_run( $tags, $content = null ) {
 				$params['where'] = $tags['where'];
 
 				if ( pods_shortcode_allow_evaluate_tags() ) {
-					$params['where'] = pods_evaluate_tags( html_entity_decode( $params['where'] ), true, '""', true );
+					$params['where'] = pods_evaluate_tags( html_entity_decode( $params['where'] ), true, '""', $pod );
```

Comment:
This is incorrect. Passing $pod would pass the pod from the shortcode, not the actual current Pod.